js/symb/leafletMap.js

```diff
@@ -112,7 +112,10 @@ class LeafletMap {
 
    defaultBounds = [];
 
-   constructor(map_id, map_options={}) {
+   /* List of GeoJson Files desired as overlays */
+   geoJSONLayers = {};
```

Comment:
The typing here seems weird. Declared as an object here but as an array in the constructor.